public/app/features/alerting/unified/hooks/useAbilities.ts

```diff
@@ -218,7 +218,12 @@ export function useAllAlertmanagerAbilities(): Abilities<AlertmanagerAction> {
       AccessControlAction.AlertingNotificationsExternalWrite
     ),
     // -- contact points --
-    [AlertmanagerAction.CreateContactPoint]: toAbility(hasConfigurationAPI, notificationsPermissions.create),
+    [AlertmanagerAction.CreateContactPoint]: toAbility(
+      hasConfigurationAPI,
+      notificationsPermissions.create,
+      // TODO: Move this into the permissions config and generalise that code to allow for an array of permissions
+      isGrafanaFlavoredAlertmanager ? AccessControlAction.AlertingReceiversCreate : null
```

Comment:
We have some cases where it isn't just one permission that allows you to see something. This felt like the cleanest way to allow checking multiple access control actions without changing a lot of existing objects to be arrays with a single element in